pages/common/pywal.md

```diff
@@ -0,0 +1,28 @@
+# pywal
+
+> A tool to create colorschemes based on dominant colors of a wallpaper.
+> Read more: https://github.com/dylanaraps/pywal.
+
+- Preview colorscheme:
+
+`wal --preview image.png`
+
+- Create colorscheme:
+
+`wal -i image.png`
+
+- Create light colorscheme:
+
+`wal -il image.png`
+
+- Skip setting wallpaper:
+
+`wal -in image.png`
+
+- Skip setting terminal colors:
+
+`wal -is image.png`
+
+- Restore colorscheme and wallpaper:
```

Comment:
Perhaps this will read better?
```suggestion
- Restore the original  color scheme and wallpaper:
```

Reply:
By "restore color scheme", it reapplies the color scheme that `wal` generated previously. E.g. in my [bspwmrc](https://github.com/kaewhyes/dotfiles/blob/master/bspwm/bspwmrc#L9), I have `wal -R`, which re-applies the color scheme, setting my polybar colors, border colors, etc.

Reply:
Would
```suggestion
- Restore colorscheme and wallpaper:
+ Restore generated color scheme and wallpaper:
```
sound better?

Reply:
Would "Restore the generated color scheme and wallpaper" sound better?

Reply:
Ah, I see. Yeah, something like that. What about `Restore the previously generated color scheme and wallpaper`?

Reply:
Instead of `Restore the original  color scheme and wallpaper:`, writing, `Restore the previously generated color scheme and wallpaper`.

Reply:
Yeah, something like that @kaewhyes